elements/pl-checkbox/pl-checkbox.py

```diff
@@ -189,6 +197,8 @@ def render(element_html, data):
             else:
                 number_correct_text = ''
```

Comment:
I think that the `number_correct_text` should be altered by the `allow-blank` flag (see https://github.com/PrairieLearn/PrairieLearn/issues/2590). The error message mentioned in the linked issue is set on line 242 here (and uses the `number_correct_text`.

Reply:
Thank you for your review!

Could the issue mentioned in #2590 please be clarified? I'm not quite sure what the bug is, since the following box seems to be quite explicit about the fact that at least one checkbox must be selected for a correct answer:
<img width="336" alt="Screen Shot 2023-01-07 at 15 53 03" src="https://user-images.githubusercontent.com/108496590/211174530-e2b1af90-85e3-474a-8a23-4ab8f180bf73.png">

As I understand, the number of correct options checked, and thus graded, is separate from `allow-blank`, which is just the attribute that allows a blank submission without the usual invalid warning.